__tests__/api.test.js

```diff
@@ -62,6 +64,47 @@ describe('GET/api/', () => {
     })
 })
 
+describe.only("GET/api/articles/:article_id", () => {
+    it("Should return an object of one article", () => {
+        return request(app)
+        .get("/api/articles/1")
+        .expect(200)
+        .then(({body}) => {
+            expect(body.article_id).toBe(1)
+            expect(body).toHaveProperty("author")
+            expect(body).toHaveProperty("title")
+            expect(body).toHaveProperty("body")
+            expect(body).toHaveProperty("topic")
+            expect(body).toHaveProperty("created_at")
+            expect(body).toHaveProperty("votes")
+            expect(body).toHaveProperty("article_img_url")
+        })
+    })
+    it("Should return an object of one article", () => {
+        return request(app)
+        .get("/api/articles/2")
+        .expect(200)
+        .then(({body}) => {
+            expect(body.article_id).toBe(2)
+            expect(body).toHaveProperty("author")
+            expect(body).toHaveProperty("title")
+            expect(body).toHaveProperty("body")
+            expect(body).toHaveProperty("topic")
+            expect(body).toHaveProperty("created_at")
+            expect(body).toHaveProperty("votes")
+            expect(body).toHaveProperty("article_img_url")
+        })
+    })
+    it("404 - Not found' when given an invalid article-id", () => {
+        return request(app)
+        .get("/api/articles/9999")
+        .expect(404)
+        .then(({body}) => {
+            expect(body.msg).toBe("Not found")
+        })
+    })
+})
```

Comment:
You're missing another sad path test for this endpoint. What if I make a request to **/api/articles/notanid**?? 👀